.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+    "rust-analyzer.check.extraEnv": {
+        "RUSTFLAGS": "--cfg rust_analyzer",

```

Comment:
Are we sure that this adds to `RUSTFLAGS` (as opposed to clobbers it?)

Reply:
Discussed offline. @djkoloski and I determined that this clobbers `RUSTFLAGS`, but that that's not particularly problematic since it should be rare for rust-analyzer to be started from a context in which the `RUSTFLAGS` environment variable is already set and the user wants its behavior preserved through rust-analyzer. E.g., it will be common for rust-analyzer to be started by an editor which is in turn run not from a shell (such as VSCode).

Reply:
Found a better way to do this by reading the RA manual. This is no longer a problem.